main.go

```diff
@@ -15,47 +17,47 @@ func printHelp() {
 		" gosplat [input dir/file] , runs directory against model\n")
 }
 
-func checkForFlags(args []string) uint8 {
-	if len(args) == 1 {
-		printHelp()
-		return earlyReturn
-	}
-	for _, arg := range args {
-		if arg == "-h" || arg == "--help" {
-			printHelp()
-			return earlyReturn
-		}
-	}
-	return continueProgram
-}
-
 var (
 	pythonPath = os.Getenv("HOME") + "/.local/share/gosplat/src/python_helper/fast_model_compare.py"
 	modelBin   = os.Getenv("HOME") + "/.local/share/gosplat/fast-fb-model.bin"
 )
 
 const (
-	earlyReturn     uint8 = 1
-	continueProgram uint8 = 2
+	earlyReturn      uint8 = 1
+	continueProgram  uint8 = 2
+	accuracyOn       uint8 = 3
+	nameSuggestionOn uint8 = 4
 )
 
-func main() {
-	if checkForFlags(os.Args) == earlyReturn {
-		return
-	}
-	dir := os.Args[1]
+// flagValues
+var (
+	dir             string
+	accuracy        int
+	help            bool
+	nameSuggestions bool
+)
+
+func checkForFlags() {
+	flag.StringVar(&dir, "d", ".", "Directory; Dir which analysis starts on")
+	flag.IntVar(&accuracy, "a", 1, "Accuracy; The amount of positives it reports on (not it will increase false positives)")
```

Comment:
Typo? Also might need a bit better explanation of what the number represents. I don't really understand and I am a developer of the tool.

Sounds like if I put the number 5 i get 5 positives. A bit confusing.

Reply:
Definitely, was thinking of changing it since this morning. Will add to it today.